src/components/next/layouts/layout/layout.tsx

```diff
@@ -516,10 +516,10 @@ const Layout = ({ children }: PropTypes) => (
         >
           Status
         </Button>,
-        <Button to={'../privacy'} style={'none'} openInNewWindow={true}>
+        <Button to={'/privacy'} style={'none'} openInNewWindow={false}>
```

Comment:
```suggestion
        <Button to={'/privacy'} style={'none'}>
```
No need to keep `openInNewWindow` if it is false

Reply:
makes sense!